recipe/meta.yaml

```
@@ -1,45 +1,82 @@
-{% set name = "<PLEASE ADD PKG NAME>" %}
-{% set version = "<PLEASE ADD PKG VERSION>" %}
+{% set name = "raphtory" %}
+{% set version = "0.13.1" %}
+{% set tests_to_skip = "test_add_properties" %}
+{% set tests_to_skip = tests_to_skip + " or test_graph_file_time_stats" %} # [osx and x86_64]
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: <sha256>
+  url: https://github.com/Pometry/Raphtory/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f99caa183387c68a3cdc0949c9000c23553cb3f33524989fa14170a3f522f69d
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  skip: True  # [py<38]
+  script:
+    - {{ PYTHON }} -m pip install ./python -vv --no-deps --no-build-isolation
+    - cargo-bundle-licenses --format yaml --output "{{ SRC_DIR }}/THIRDPARTY.yml"
+  # gql-with-all not on s390x
+  skip: True  # [py<38 or s390x]
 
 requirements:
+  build:
+    # C Compiler is needed on linux to get missing libgcc-ng as a run_export
+    - {{ compiler('c') }}     # [linux]
+    - {{ compiler('rust') }}
+    - cmake
+    - cargo-bundle-licenses
   host:
     - python
     - pip
+    - protobuf
```

```suggestion
    - protobuf 4.25.3
```
[>=4.25.3,<5](https://github.com/Pometry/Raphtory/blob/85a9eab8839cc2e602f2737897e48de8653ad967/.environment.yml#L6) just to be safe

---

That's coming from a different part of the tree, though, isn't it?  We just want the `python/*` parts?

---

True, but it's still better to pin it down to ensure reproducibility of build in the future. That's a safe approach with low lever libraries such as `protobuf`.

---

It probably won't hurt to pin it, done in [e320ba7](https://github.com/AnacondaRecipes/raphtory-feedstock/pull/1/commits/e320ba7296553d8083f06b32111ad6f1d1d82618).